source/sass/blocks/advantages.scss

```
@@ -0,0 +1,41 @@
+.advantages__wrapper {
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+.advantages__headline,
+.advantages__caption {
+  margin: 0;
+  padding: 0;
+  font-weight: 300;
+  font-size: 24px;
+  line-height: 120%;
+  color: $text-base;
+  text-align: center;
+}
+
+.advantages-wrap {
+  background-image: url("../../img/advantages/grey-background.jpg") no-repeat;
```

```suggestion
  background-image: url("../img/advantages/grey-background.jpg") no-repeat;
```
Структура проекта:
```bash
site.ru/
  css/
    style.css
  img/
    test.jpg
  catalog.html
  index.html
```
Файл стилей лежит в каталоге **css**. Чтобы из этого файла получить доступ к изображению в каталоге **img** браузеру нужно перейти на один уровень вверх (символы `../`), а если 2 раза указать эти символы, то браузер должен перейти на 2 уровня вверх, то есть за пределы корня сайта, а это невозможно.

Про относительные пути есть [статья](https://up.htmlacademy.ru/adaptive/22/book/2.3) в учебнике.